var/spack/repos/builtin/packages/py-lazy/package.py

```diff
@@ -11,6 +11,8 @@ class PyLazy(PythonPackage):
 
     pypi = "lazy/lazy-1.2.zip"
```

Comment:
They switched to .tar.gz, did you actually test to make sure this installs? You'll need to add a url_for_version function